README.md

```diff
@@ -381,3 +385,9 @@ This has been fixed with version `1.1.0` by falling back to letting the agent fi
 **Customer question**: We have `proxy_set_header` directives only in server blocks in the NGINX configuration. Do we have to set `opentracing_propagate_context` for all locations?
 
 **Answer**: No, it can be set at main (http), server, and location level. So just add it to the server blocks where you have the `proxy_set_header` directives.
+
+### Authorization Subrequests are not traced
+
+The subrequest tracing requires the configuration entry `log_subrequest on;` at main (`http`) level.
+Versions before `1.1.2` cause log spam without that setting.
+At least version `1.2.0` is required in order to show the correct URI for subrequests.
```

Comment:
`nitpick` "At least version `1.2.0` is required" sounds a bit bumpy. Maybe something like "From version `1.2.0` onwards traces from subrequest show the correct URI.
```suggestion
From version `1.2.0` onwards traces from subrequests show the correct URI.
```

Reply:
Suggestion applied. Thanks.